### ISSUE TITLE
Audio & Video works well on same network but not in different network

### ISSUE THREAD

**Issue by shariful2011** [shanet/WebRTC-Example#14]
I tried your source which is really a good one. But I am facing one problem. On Same netwrok it works perfectly but on different network remote side video & audio is not working. My server is a dedicated Windows 2012 server with static IP. On Same network means if I use it on a PC and laptop with wifi on the same network it works perfectly but when ever I go to different network it doesn't work. What should I do?

**Comment by shanet** [comment]
This usually means there is a firewall or something else in the middle that is preventing a connection from being made successfully. WebRTC uses ports other than the usual 80 and 443 so nmap is your friend here to check if the necessary ports are open. Otherwise, you can use a TURN server.

**Comment by shariful2011** [comment]
I tried with http://xirsys.com STUN & TURN server. This is what I received:
1. If I clicked start video from first opened link I got error on second one as following:
Failed to parse SessionDescription.  Failed to parse audio codecs correctly.
Error processing ICE candidate

2. If I clicked start video from second opened link I got both description and addstream on both side but no remote video on both side.
3. After that I clicked start video from second open link I received Error processing ICE candidate on first opened link.

Also I used port 443. On same network both side video/audio is showing but not in different network.
